azure-postgresql-flexible-server.yml

```diff
@@ -0,0 +1,192 @@
+  - field_name: sku_name
+    type: string
+    details: Azure sku (typically, tier [B,GP,MO] + family [Gen4,Gen5] + cores, e.g. B_Gen4_1, GP_Gen5_8, see https://docs.microsoft.com/en-us/azure/mysql/concepts-pricing-tiers) Will be computed from cores if empty.
+    default: ""
+  computed_inputs:
+  - name: labels
+    default: ${json.marshal(request.default_labels)}
+    overwrite: true
+    type: object
+  template_refs:
+    data: terraform/azure-postgresql-flexible-server/provision/data.tf
+    main: terraform/azure-postgresql-flexible-server/provision/main.tf
+    outputs: terraform/azure-postgresql-flexible-server/provision/outputs.tf
+    provider: terraform/azure-postgresql-flexible-server/provision/provider.tf
+    variables: terraform/azure-postgresql-flexible-server/provision/variables.tf
+    versions: terraform/azure-postgresql-flexible-server/provision/versions.tf
+  outputs:
+  - field_name: name
+    type: string
+    details: The name of the database.
+  - field_name: hostname
+    type: string
+    details: Hostname or IP address of the exposed PostgreSQL endpoint used by clients to connect to the service.
+  - field_name: port
+    type: integer
+    details: The port number of the exposed PostgreSQL instance.
+  - field_name: username
+    type: string
+    details: The username to authenticate to the database instance.
+  - field_name: password
+    type: string
+    details: The password to authenticate to the database instance.
+bind:
+  plan_inputs: []
+  user_inputs: []
+  computed_inputs:
+  - name: db_name
+    type: string
+    default: ${instance.details["name"]}
+    overwrite: true
+  - name: hostname
+    type: string
+    default: ${instance.details["hostname"]}
+    overwrite: true
+  - name: port
+    type: integer
+    default: ${instance.details["port"]}
+    overwrite: true
+  - name: admin_username
+    type: string
+    default: ${instance.details["username"]}
+    overwrite: true
+  - name: admin_password
+    type: string
+    default: ${instance.details["password"]}
+    overwrite: true
+  template_refs:
+    data: terraform/azure-postgresql-flexible-server/bind/data.tf
+    main: terraform/azure-postgresql-flexible-server/bind/main.tf
+    outputs: terraform/azure-postgresql-flexible-server/bind/outputs.tf
+    provider: terraform/azure-postgresql-flexible-server/bind/provider.tf
+    variables: terraform/azure-postgresql-flexible-server/bind/variables.tf
+    versions: terraform/azure-postgresql-flexible-server/bind/versions.tf
+  outputs:
+  - field_name: username
+    type: string
+    details: The username to authenticate to the database instance.
+  - field_name: password
+    type: string
+    details: The password to authenticate to the database instance.
+  - field_name: uri
+    type: string
+    details: The uri to connect to the database instance and database.
+  - field_name: jdbcUrl
+    type: string
+    details: The jdbc url to connect to the database instance and database.
+examples:
```

Comment:
Nice. I've added a task to make sure this gets added to CI so it's actually run.